requirements_windows.txt

```diff
@@ -5,6 +5,10 @@ h11
 websockets==8.*
 wsproto==0.13.*
 
+# Packaging
+twine
+wheel
+
```

Comment:
Probably out of scope of this PR but - is there a reason this Windows-specific requirements file exists, instead of using a single requirements file with environment markers such as `some-dep; platform="win32"`?

Reply:
this is where I'm not confortable and mix probably things,

I originally thought once https://github.com/encode/uvicorn/pull/666 land that we could avoid having separate `requirements.txt` for win and linux , that's probably what you imply, the use of specific os markers inside the setup.py ?

but now I htink about it we may still need both requirements files because they're just used by the CI and specifically the install script

again, I may be totally off, happy to learn somehting today !

Reply:
oula ok yes now I get it, didn't remember ou could do that in the requirements.txt too

Reply:
will do another PR on top of this one and we can have the install script back were it was